canonical-web-and-design/canonical.com
# Add filters & sorting to career pages

## Done
- Add filters to all careers pages

## QA
- Run the website locally using `./run`
- Go to [/careers/all](http://0.0.0.0:8002/careers/all) and click on the `Availabl roles` tab
- Change `Filter by` from the top of the page and check if the jobs are filtered
- Change `Sort by` and make sure the list is sorted accordingly 

## Issue

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1511
